package.json

```diff
@@ -17,14 +17,14 @@
     "@bcrs-shared-components/action-chip": "1.0.6",
     "@bcrs-shared-components/bread-crumb": "1.0.1",
     "@bcrs-shared-components/certify": "1.0.18",
-    "@bcrs-shared-components/confirm-dialog": "1.0.1",
-    "@bcrs-shared-components/contact-info": "1.0.29",
+    "@bcrs-shared-components/confirm-dialog": "1.0.3",
```

Comment:
Should this be v1.1.0 ?

Reply:
Doh, yes to this and the Q below 👍 
Updated!